forwardProp/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "goober-forward-prop",
+  "amdName": "gooberForwardProp",
+  "version": "0.0.1",
+  "description": "The forwardProp addon function for goober",
+  "sideEffects": false,
+  "main": "dist/goober-forward-prop.js",
+  "module": "dist/goober-forward-prop.mjs",
+  "umd:main": "dist/goober-forward-prop.umd.js",
+  "source": "src/index.js",
+  "unpkg": "dist/goober-forward-prop.umd.js",
+  "types": "./forward-prop.d.ts",
+  "scripts": {
+    "build:lib": "rm -rf dist && microbundle --entry src/index.js --name gooberForwardProp --no-sourcemap",
+    "test": "jest --setupFiles ./jest.setup.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cristianbote/goober.git",
+    "directory": "forwardProp"
+  },
+  "author": "Cristian <botecristian@yahoo.com>",
```

Comment:
While this indeed is under my user's scope, this package is all on you @devrnt! 😃  So, I would encourage you to stamp your name in here! 🎉

Reply:
Okay changes noted 😀